internal/cmd/api-key/command_list.go

```diff
@@ -153,7 +153,9 @@ func getServiceAccountsMap(serviceAccounts []iamv2.IamV2ServiceAccount) map[stri
 func getUsersMap(users []*ccloudv1.User) map[int32]*ccloudv1.User {
 	userMap := make(map[int32]*ccloudv1.User)
 	for _, user := range users {
-		userMap[user.Id] = user
+		if user != nil {
```

Comment:
This check seems redundant given the other check made in this PR.

Reply:
I'd rather use the getter syntax than adding defensive `if` statements everywhere.

Reply:
It's mostly redundant, but it's to preempt another edge case: if the backend returns a `nil` user as part of the list.

Reply:
Ok I removed the if. It still has the getter function.